connector/doc/guides/jobrunner.rst

```diff
@@ -25,9 +25,12 @@ Intermediate subchannels which are not configured explicitly are autocreated
 with an unlimited capacity (except the root channel which if not configured gets
 a default capacity of 1).
 
+A delay between jobs can be set at the channel level with the ``delay`` key.
```

Comment:
Just wondering if `wait` could not be more appropriate, since `delay` is also the name of the method to delay a job?